src/Codeception/Module/Laravel5.php

```diff
@@ -114,7 +114,7 @@ public function __construct(ModuleContainer $container, $config = null)
             [
                 'cleanup' => true,
                 'run_database_migrations' => false,
-                'database_migrations_path' => '',
+                'database_migrations_path' => 'database/migrations',
```

Comment:
I would suggest to use Laravel approach (see example in [NotificationTableCommand](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Notifications/Console/NotificationTableCommand.php#L77)) rather than a static value.
Example:
```php
$this->config['database_migrations_path'] = $this->app->databasePath().'/migrations';
```